src/main/drivers/bus_i2c_stm32f10x.c

```diff
@@ -87,6 +87,8 @@ void I2C2_EV_IRQHandler(void)
     i2c_ev_handler();
 }
 
+// I2C_DEFAULT_TIMEOUT already defined in bus_i2c.h
```

Comment:
this is only hiding possible problem ... one of the `#define`s should be removed ..